### PR TITLE
docs: add script to push latest docs from cli repo to the website

### PR DESCRIPTION
**Pull request by activus-d**
**Description**
This pull request creates the workflow to push latest docs from the cli repo to asyncapi.com/docs

**Related issue(s)**
This pull request is a continuation from [Docs: add CLI docs to tools content bucket #1078](https://github.com/asyncapi/website/issues/1078) and also resolves issue [Integrate CLI docs with asyncapi website #448](https://github.com/asyncapi/cli/issues/448)

### CODE REVIEW

File: .github/workflows/update-docs-in-website.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-cli-docs-${{ github.sha }}
+      - name: Copy cli folder from Current Repo to Another
+        working-directory: ./website
+        run: |
+          mkdir -p ./pages/docs/tools/cli
+          rm ../cli/docs/README.md
+          rm -r ../cli/docs/jsdoc2md-handlebars
```
Comment (derberg) [inline .github/workflows/update-docs-in-website.yml:40]: there is no readme and no handlebars docs to remove, so these lines can be removed

File: .github/workflows/update-docs-in-website.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          printf "%s\ntitle: cli\nweight: 3\n%s" "---" "---"> ../cli/docs/_section.md
```
Comment (derberg) [inline .github/workflows/update-docs-in-website.yml:41]:
```suggestion
          printf "%s\ntitle: CLI\nweight: 30\n%s" "---" "---"> ../cli/docs/_section.md
```

let's increase weight so CLI is next on the list after generator, but if needed, there is a lot of other stuff that can  be placed between these 2 in navigaton

File: .github/workflows/update-docs-in-website.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-cli-docs-${{ github.sha }}
+      - name: Copy cli folder from Current Repo to Another
+        working-directory: ./website
+        run: |
+          mkdir -p ./pages/docs/tools/cli
+          printf "%s\ntitle: cli\nweight: 4\n%s" "---" "---"> ../cli/docs/_section.md
```
Comment (derberg) [inline .github/workflows/update-docs-in-website.yml:39]: I made code suggestions to this line but they are not applied

Comment (activus-d) [reply .github/workflows/update-docs-in-website.yml:39]: oh yes! I increase the weight from `3` to `4` how high would you suggest the weight to be?

Comment (derberg) [reply .github/workflows/update-docs-in-website.yml:39]: did you see https://github.com/asyncapi/cli/pull/476#discussion_r1143379055 comment?

Comment (activus-d) [reply .github/workflows/update-docs-in-website.yml]: @derberg I've made pushed a new commit

Comment (derberg) [reply .github/workflows/update-docs-in-website.yml]: @activus-d I think you missed https://github.com/asyncapi/cli/pull/476#discussion_r1143379055 to make title uppeer case `CLI` and not `cli`

Comment (activus-d) [reply .github/workflows/update-docs-in-website.yml]: @derberg I've pushed a new commit to implement the review

**Review by derberg — APPROVED** [review]
ok, looks like we have all needed 💪🏼 
Thanks @activus-d

**Comment by derberg** [comment]
@activus-d now to make sure it works, we need to make some changes in `docs` folder. You can open a PR that updates the `usage` markdown document and above `COMMANDS` add:

```
TOPICS
  config    CLI config settings
  generate  Generate models and template
  start     Start AsyncAPI studio
```

**Comment by activus-d** [comment]
> to make sure it works, we need to make some changes in `docs` folder. You can open a PR that updates the `usage` markdown document and above `COMMANDS` ad

@derberg I have made the required changes and made a pull request

**Comment by derberg** [comment]
@activus-d look, automation worked like a charm -> https://github.com/asyncapi/website/pull/1591/files

**Comment by activus-d** [comment]
> @activus-d look, automation worked like a charm -> https://github.com/asyncapi/website/pull/1591/files

@derberg Whooho🕺🕺🕺! So amazing to see🎉🎉🎉

**Comment by derberg** [comment]
ok, but something got wrong, in preview only one doc is visible https://deploy-preview-1591--asyncapi-website.netlify.app/docs/tools/cli/usage 🤔

**Comment by activus-d** [comment]
> ok, but something got wrong, in preview only one doc is visible https://deploy-preview-1591--asyncapi-website.netlify.app/docs/tools/cli/usage 🤔

@derberg I can see that. The index and the installation docs are missing. That's strange

**Comment by derberg** [comment]
@activus-d I think it is website bug, investigating

**Comment by derberg** [comment]
@activus-d ok, got the fix, do you want to fix it? small PR to CLI repo. I can explain

**Comment by derberg** [comment]
actually I opened a PR https://github.com/asyncapi/cli/pull/525 as there were bit more things to fix

**Pull request merged**